lib/stringify.js

```diff
@@ -15,6 +15,14 @@ var arrayPrefixGenerators = {
     }
 };
 
+var pushToArray = function (arr, valueOrArray) {
+    if (Array.isArray(valueOrArray)) {
+        Array.prototype.push.apply(arr, valueOrArray);
```

Comment:
Why is it bad to depend on `Array#push` and `apply`? They're fairly standard :). It'd look nicer with array spread, of course.

I had no idea about `Symbol.isConcatSpreadable`. As far as I can tell, the operand will always be an array or a string, though.

Reply:
They're all standard, but the more things that the package depends on existing, the more likely it will break if people mess with the builtin prototypes.

Reply:
That's technically correct, but is it really worth worrying about? It seems unlikely that anyone will overwrite ES3 methods from prototypes in this day and age? Especially methods that have been in the language for 20 years?

I'm caught a bit off-guard because I haven't heard that kind of reasoning for preferring one piece of code over another for a very long time. I'm aware of MooTools, should.js and some of the even crazier things people did before, but hasn't the community stopped doing it and moved on at this point?

Reply:
This is the web; I always write all my code assuming that everything will be deleted off the prototype (or replaced with a malicious version) after my code runs :-)

This code predates me, and your PR is changing the attack surface. I agree it's a minor concern in practice, but it's still nonzero.